Extensions/PathfindingBehavior/pathfindingruntimebehavior.ts

```diff
@@ -685,12 +686,12 @@ namespace gdjs {
 
         //No so construct a new node (or get it from the cache)...
         // @ts-ignore
-        let newNode: gdjs.PathfindingRuntimeBehavior.Node = null;
+        let newNode: Node | undefinied;
```

Comment:
I generally prefer to keep null rather than undefined, as null is explicit (this was set to null to signify that there is no object), rather than undefined (which is set even if you don't write it, and so which indicates much more often an error whenever you see it).